src/index.js

```diff
@@ -1,37 +1,66 @@
 const { SQSClient, SendMessageCommand } = require('@aws-sdk/client-sqs')
-const { logger, serializeError } = require('./logging')
+const { SNSClient, PublishCommand } = require('@aws-sdk/client-sns')
+const { logger: defaultLogger, serializeError } = require('./logging')
+const config = require('./config')
+const { createS3URL } = require('./s3')
+const { createClientConfig } = require('./aws')
 const {
   SQS_INDEXER_QUEUE_URL: indexerQueue,
   SQS_INDEXER_QUEUE_REGION: indexerQueueRegion
 } = process.env
 
-const SQSclient = new SQSClient({
+const defaultSQSClient = new SQSClient({
   region: indexerQueueRegion
 })
 
 /**
- * @type {import('aws-lambda').SNSHandler}
+ * Create the main function that will handle SNS events
+ * @param {object} arg
+ * @param {string} arg.eventsTopic - events pubsub topic - e.g. an AWS SNS TopicArn
+ * @param {import('@aws-sdk/client-sns').SNSClient} [arg.eventsTopicClient] - client to use to send commands to eventsTopic
```

Comment:
Types for the other two params? 🙏

Reply:
I think if you do it should be a separate PR, and we should decide if we want that with the Nearform guys since this is essentially their responsibility.

Reply:
i fixed these params

Reply:
also others can weigh in on https://github.com/elastic-ipfs/bucket-to-indexer/issues/14

Reply:
I'm also for a separate PR for eslint rule.